### ISSUE TITLE
Can't command line addNote w/o noteText (just attachment)

### ISSUE THREAD

**Issue by herrlich98** [baumgarr/nixnote2#271]
nixnote2-daily+20170205  (on Ubuntu 16.04.2 LTS)

nixnote2 addNote --attachment=file.pdf --title="test note"
will not create a note (never see "NNNNN has been created").

...and, let me know if I should break this into two issues, --noteText="" also doesn't work but --noteText=" " (single space) does work but then gets an error sync'ing...

ERROR 2017-02-12 17:20:40.075 ( communication/communicationmanager.cpp @ 557 ) EDAMUserException: 11 
ERROR 2017-02-12 17:20:40.075 ( threads/syncrunner.cpp @ 1149 ) "Error uploading note:test not #15" 
 INFO 2017-02-12 17:20:40.075 ( nixnote.cpp @ 1875 ) "Unable to update note.  Invalid note structure : :EDAMUserException: ENML_VALIDATION parameter=Element type \"en-media\" must be followed by either attribute specifications, \">\" or \"/>\"." 
 INFO 2017-02-12 17:20:40.075 ( nixnote.cpp @ 1875 ) "Unable to update note.  Invalid note structure : :EDAMUserException: ENML_VALIDATION parameter=Element type \"en-media\" must be followed by either attribute specifications, \">\" or \"/>\"." 

I have tried this with a few difference pdf's so I don't think it is related to the pdf.

**Comment by baumgarr** [comment]
Should be fixed. Editing the note will be a temporary solution. You just need to put something like a space in the note and it will be corrected after you save it.

The documentation isn't very clear, but it is also expecting note text. For example, 

nixnote2 addNote --noteText="attachment test %%" --attachment=test.pdf

It wants some type of attachment point in a note. The %% tells it where to put the attachment. For a note with no text you should put noteText="%%"

**Comment by herrlich98** [comment]
Thanks for so promptly replying to my newbie question.

This morning I just updated to "nixnote2-daily+20170212"

Adding %% does seem to work pretty well for adding the note but I still have the syncing error until I edit it via the GUI.  I tried doing a cli appendNote to kinda do an "edit" via the cli but that didn't fix the syncing error either.  It doesn't look like I can do a cli addNode with --attachment purely with the cli even using %%  I did a pure text addNote and that worked and synced correctly.
